src/spyglass/utils/dj_mixin.py

```diff
@@ -262,30 +260,27 @@ def _import_merge_tables(self):
         )
```

Comment:
Because of how delete_downtream_parts finds part tables, should the `Group` tables `PositionGroup` and `SortedSpikesGroup` be imported here to in order to have them in the graph when found?

Reply:
I added these items and renamed the func accordingly